docs/user/guides/creating-config-file.rst

```diff
@@ -0,0 +1,100 @@
+================================
+Creating your configuration file
+================================
+
+As part of the initial setup for your Read the Docs site, you need to create a 
+**configuration file** called ``.readthedocs.yaml``.
+
+The configuration file like a personalized guidebook that tells the platform 
+what specific settings for your project. Every project is unique, and by using 
+a configuration file, you can tailor the behavior of Read the Docs to match 
+your specific needs.
+
+This how-to guide covers
+#. Where to put your configuration file
+#. The required parts of your configuration file
+
+This should be enough to get you started!
+
+.. tip::
+   
+   The complete list of all possible ``.readthedocs.yaml`` settings, including 
+   the optional settings not covered in on this page, is found on 
+   :doc:`the configuration file reference page </docs/user/config-file/index.rst>`.
+
+####################################
+Where to put your configuration file
+####################################
+
+The ``.readthedocs.yaml`` file should be placed in the **root directory**. The 
+root directory is the top-most location of your repository.
+
+To make the file, navigate to your root directory and create a new 
+file with the name ``.readthedocs.yaml``.
```

Comment:
```suggestion
Add a new file with the exact name ``.readthedocs.yaml`` in the repository's root directory.
We will get to the contents of the file in a moment.
```

I think people have preliminary knowledge of adding new files to a repository.

I wonder if we can skip the _explanation_ of why the config file is hidden? It breaks apart the creation of the file from the content, it would perhaps be easier for the user to have both instructions at the same time.